extensions/vscode/lsp/.gitignore

```diff
@@ -0,0 +1,4 @@
+out
+node_modules
+client/server
+.vscode-test
```

Comment:
client/server?
This seems inaccurate.

We should create a .gitignore ourselves after creating the vsix build rule and seeing what files `git status` reports as cruft after that completes.

Reply:
Okay.